### DISCUSSION TITLE
config.ini documenation

### DISCUSSION THREAD

**Discussion by jm-daemon** [profi200/open_agb_firm#27] [Q&A]
Can we get a description of how the settings work in the new config.ini?

**Comment by profi200** [comment]
I was planning on adding a bit of documentation later. At least for what's available to configure right now it's fairly self-explanatory and i write out all config options to the file by default (later i will only do so for the most used/safe ones). Keep in mind the config format (as in the available options) is nowhere near finished. There will be several changes until the first non-alpha release.

**Reply by jm-daemon** [reply]
What are the allowed values? The brightness isn't so hot on my 3ds xl and when I increased the brightness value the backlight turned off.

**Reply by profi200** [reply]
0 to 64. The later is a hard limit for now for safety. Proper brightness control has not been implemented yet. Unfortunately it's actually possible to give the backlight too much juice by setting this too high hence why "safety".
